openshift/README.md

```diff
@@ -57,7 +57,13 @@ oc create -n $NAMESPACE configmap $APP_NAME-server-config \
   --from-literal=SERVER_PORT=8080
 ```
 
+*Note:* We use the NRS object storage for CHEFS. For related information, see:
+
+- Environment variables created by the following config map can be overridden by your [Object Storage Configuration](../app/config/test.json)
```

Comment:
Not sure if there's any value linking to the test.json as that specific file is only used in CI/CD environments. Instead, there should be a note saying create `local.json` and add in your values there or something along those lines.